### PR TITLE
chore: edit github pr template

### PR DESCRIPTION
**Pull request by kevinpagtakhan**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Description
+<!---
+Thanks for contributing to the Amplitude JavaScript SDK! 🎉
```
Comment (yuhao900914) [inline .github/pull_request_template.md:2]: The same question about the pr template. 
Not limited to javascript sdk.

Comment (kevinpagtakhan) [reply .github/pull_request_template.md:2]: Yeah since there is only one, i think we should name this with something more generic

Comment (yuhao900914) [reply .github/pull_request_template.md:2]: Agree.

Comment (kevinpagtakhan) [reply .github/pull_request_template.md]: Alright i just said "Amplitude TypeScript repository"

**Review by dnshi — APPROVED** [review]

**Pull request merged**